docs/samples/git-setup/clustertemplates/hardwaretemplates/sno-ran-du/sno-ran-du-v1.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: sno-ran-du-template-configmap-v1
```

Comment:
The hardware templates are independent from the cluster templates and will likely be shared between them, so they should not share the same name. The name should be something like dell-intel-du-template-configmap-v1 I think.

Reply:
Agreed, that was not very well chosen. I would like to keep it generic here and just go with `placeholder-du-template-configmap-v1`